model/config.go

```diff
@@ -519,11 +519,6 @@ func (o *Config) SetDefaults() {
 		o.FileSettings.AmazonS3Endpoint = "s3.amazonaws.com"
 	}
 
-	if o.FileSettings.AmazonS3Region == "" {
-		// Defaults to "us-east-1" region.
-		o.FileSettings.AmazonS3Region = "us-east-1"
```

Comment:
No if region is blank minio will use GetBucketLocation to get the appropriate region, if that fails then we look for x-amz-bucket-region, if that fails then we default to `us-east-1`